files/check_hddtemp

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Can you add a comment at the beginning describing what the check checks and how it's called? I know it's documented further down but I think it's a lot faster to just have it on top. Maybe just paste `--help` there. Also you can get rid of that emacs garbage there